joinmarket/wallet.py

```diff
@@ -147,9 +147,10 @@ def __init__(self,
             self.index.append([0, 0])
 
     def read_wallet_file_data(self, filename, pwd=None):
+        dirname = os.path.split(os.path.abspath(__file__))
         self.path = None
         self.index_cache = [[0, 0]] * self.max_mix_depth
-        path = os.path.join('wallets', filename)
+        path = os.path.join(dirname[0][:-11], 'wallets', filename)
```

Comment:
Using `-11` and split here is fragile.  Probably better to do something like:
```
dirname = os.path.dirname(os.path.abspath(__file__))
wallet_path = os.path.join(dirname, '..', 'wallets', filename)
```

Although I think moving wallet and log dir to configuration variables would probably be better still